R/module_nested_tabs.R

```diff
@@ -92,22 +92,22 @@ ui_teal_module.teal_module <- function(id, modules, depth = 0L) {
   args <- c(list(id = ns("module")), modules$ui_args)
 
   ui_teal <- tagList(
-    div(
-      id = ns("validate_datanames"),
-      ui_validate_reactive_teal_data(ns("validate_datanames"))
-    ),
     shinyjs::hidden(
       tags$div(
-        id = ns("transformer_failure_info"),
+        id = ns("transform_failure_info"),
         class = "teal_validated",
         div(
           class = "teal-output-warning",
-          "One of transformers failed. Please fix and continue."
+          "One of transforms failed. Please fix and continue."
```

Comment:
transform - verb (to transform)
transforms - verb (he transforms)
transformer - noun (he is a transformer)

Reply:
We can also call it `transformations`, even though it sounds long

Reply:
f0b87f8fd9ffa01804fd416294889b984ee53d5b

Reply:
I left `transform` as a verb in function names, but used `transformator` and `transformators` in the documentation